[GSoD 2021] Added OpenAPI learning resources

iamrajiv opened the pull request:
Related: #1012

![Screenshot 2021-06-19 at 5 50 10 PM](https://user-images.githubusercontent.com/42106787/122642206-d9ebbe00-d126-11eb-9102-84f8c4007277.jpg)
![Screenshot 2021-06-19 at 5 50 20 PM](https://user-images.githubusercontent.com/42106787/122642208-dc4e1800-d126-11eb-832b-c12596fd9603.jpg)

proudofsimin reviewed (changes requested):
Hi Rajiv,
Can you explain how you organize the sequence of these resources? It seems just random materials putting together, and most of them are from the same website. If you are only providing some relevant information about OpenAPI, the resources are better to cover different aspects (or from different organizations). On the other hand, if you are guiding readers what an OpenAPI is (as I saw you put the tutorials in), maybe you could consider re-organize the topics a little bit (from basic to advanced). Also a short description of your purpose of this page is required at the beginning.
Btw, please remove the link to each topic. You don't need to link them twice.

iamrajiv commented:
Hi, @proudofsimin thank you for the review and suggestion. I have structured the learning resources section and removed the duplicate links. I have provided learning resources according to Wechaty user requirements and which are not covered in PR [#917](https://github.com/wechaty/wechaty.js.org/pull/917). I have added these learning resources at the end because that will be more suitable since after the merge of PR [#917](https://github.com/wechaty/wechaty.js.org/pull/917) other things will be added in front of the document that covers basic and general learning about OpenAPI. If a user wants to learn more I have added the relevant links in additional resources they can go through it.

lijiarui commented on docusaurus/docs/polyglot/openapi/overview.md:
```diff
@@ -26,6 +26,41 @@ Read blogs with the `openapi` tag at <https://wechaty.js.org/tags.html#openapi>
 ## History
 
+## Learning Resources
+
```
> It seems just random materials putting together, and most of them are from the same website. If you are only providing some relevant information about OpenAPI, the resources are better to cover different aspects (or from different organizations). On the other hand, if you are guiding readers what an OpenAPI is (as I saw you put the tutorials in), maybe you could consider reconstruct the topics a little bit (from basic to advanced). Also a short description of your purpose of this page is required at the beginning.

I agree with @proudofsimin 

I think we may more information from README in [wechaty openapi official repo](https://github.com/wechaty/openapi), content in README is more relative to wechaty docs.

We can have some learning resources, but I don't think in a wechaty openai introduction page, 2/3 of the content is introducing resources, it looks a bit overwhelming.

iamrajiv replied:
@lijiarui I have removed unnecessary resources and added only relevant resources about OpenAPI.

lijiarui replied:
Hi @iamrajiv Thanks for removing unnecessary resources,  I think we may add more information from README in [wechaty openapi official repo](https://github.com/wechaty/openapi), content in README is more relative to wechaty docs.

iamrajiv replied:
@lijiarui Yeah I made a separate PR for that https://github.com/wechaty/openapi/pull/18. This PR is an addition to https://github.com/wechaty/wechaty.js.org/pull/917 so that people can get an overview on the website too about learning resources of OpenAPI.

lijiarui replied:
great! I think it lgtm.

huan commented:
Thank you very much for adding resources to the open API docs!
